docs/server-side-rendering.md

```diff
@@ -20,6 +20,29 @@ React components used on the server will be rendered to HTML by being passed to
 * Libraries that are used on the server should be mindful of the DOM not being available on the server, and should either: be modified to work without the DOM; have non-DOM specific fallbacks; or fail in an obvious manner.
 * Singletons should be avoided, as once instantiated they will persist for the entire duration of the `node` process.
 
+### Caching
+
+Because it is necessary to serve the redux state along with a server-rendered page, we use two levels of cache on the server—one to store raw query data, from which we can generate and serve redux state, and one to store rendered layouts.
+
+##### Data Cache
+
+Caching data is currently left to the controller for a [given](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/themes/controller.jsx) [section](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/theme/controller.jsx). Request timestamps are used to force expiration.
```

Comment:
I've started using relative links in [`routing.md`](https://github.com/Automattic/wp-calypso/blob/09ba6bacb8f54d659643ce7f27dc0d5ecd586c08/docs/routing.md) (i.e. using `../client` instead of the entire URL in this case), which works quite well. I imagine they can be useful in an editor or IDE that supports them. (There are a couple of other instances below.)